### PR TITLE
Update atp-safe-links.md

### PR DESCRIPTION
**Pull request by RaduTudorMS**
- Specifying that URLs remain rewritten only for manual forwarding
- Update regarding how safe links behaves in relation to automatic forwarding/redirect

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/atp-safe-links.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ The following table describes scenarios for Safe Links in Microsoft 365 and Offi
 
 Safe Links scans incoming email for known malicious hyperlinks. Scanned URLs are rewritten using the Microsoft standard URL prefix: `https://nam01.safelinks.protection.outlook.com`. After the link is rewritten, it's analyzed for potentially malicious content.
 
-After Safe Links rewrites a URL, the URL remains rewritten, even if the message is forwarded or replied to. Additional links that are added to the forwarded or replied to message are not rewritten.
+After Safe Links rewrites a URL, the URL remains rewritten, if the message is manually forwarded or replied to (both to internal and external recipients). Additional links that are added to the forwarded or replied to message are not rewritten. However in the case of automatic forwarding/redirect (Ex: Inbox rules, SMTP forwarding), the URL will not be rewritten for the email copy intended for the final recipient, unless that recipient is also protected by Safe Links, or the URL had already been rewritten in a previous communication.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/atp-safe-links.md]:
```suggestion
After Safe Links rewrites a URL, the URL remains rewritten if the message is manually forwarded or replied to (both to internal and external recipients). Additional links that are added to the forwarded or replied to message are not rewritten. However, in the case of automatic forwarding/redirect (for example Inbox rules and SMTP forwarding), the URL will not be rewritten for the email copy intended for the final recipient unless that recipient is also protected by Safe Links or the URL had already been rewritten in a previous communication. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@RaduTudorMS

**Pull request merged**